Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM drydock-prod.workiva.net/workiva/dart_build_image:1
+FROM google/dart:2
```

Comment:
Do we still need this Dockerfile? If we do, we could at least delete the `ARG BUILD_ARTIFACTS_DART_PUBSPECLOCK` line below as it's no longer needed.

Reply:
I left the dockerfile for now but removed the ARG that isn't needed

Reply:
fwiw, my understanding is that we do need the Dockerfile to at least run a `pub get` so that RM can do their dependency tracking/auditing.